astro/src/content/docs/extend/code/lambdas/index.mdx

```diff
@@ -4,6 +4,7 @@ description: An overview of the lambda support of FusionAuth
 section: extend
 subcategory: code
 tertcategory: lambdas
+topOfNav: true
```

Comment:
Nothing wrong with this code.  In general this page title does not follow the same format as customization.  This only has Lamdas where the Customization has 'Overview' appended.

![Screenshot 2023-11-17 at 11 18 47 AM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/e46ffd70-6aef-4dfc-bc81-5fbb627aa1e9)

![Screenshot 2023-11-17 at 11 20 41 AM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/b90199e8-dfac-4290-8660-b13225660d01)

Reply:
Yes, the titles are a hot mess and not consistent. I fixed a few where they sprawled across the page, but we should definitely make them consistent w/r/t title casing and whether to put 'Overview' on it or not.

Not sure that should go into this PR.